loop/storage/redis.js

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 "use strict";
-var redis = require("redis");
+var redis = require("./redis_client");
```

Comment:
I wish we had a configuration for this to let us use the default redis for real redis server and the custom redis_client for proxified shard.